Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.h

```diff
@@ -106,7 +107,7 @@ class ITK_TEMPLATE_EXPORT BSplineInterpolationWeightFunction
 
 #if !defined(ITK_LEGACY_REMOVE)
   /** Get support region size. */
-  itkLegacyMacro(SizeType GetSupportSize() const) { return Self::SupportSize; };
+  itkLegacyMacro(SizeType GetSupportSize() const) { return SizeType::Filled(VSplineOrder + 1); };
```

Comment:
@hjmjohnson Now that you have defined `SupportSize` in the proper C++14 compatible way, is it still necessary to adjust `GetSupportSize()` like you propose here? If technically possible, I still prefer a simple `return Self::SupportSize`.